src/covid_model_seiir_pipeline/paths.py

```diff
@@ -8,6 +8,9 @@
 from covid_shared.shell_tools import mkdir
 from loguru import logger
 
+DRAW_FILE_TEMPLATE = 'draw_{draw_id}.csv'
+LOCATION_FILE_TEMPLATE = 'location_{location_id}.csv'
```

Comment:
I might be missing something but I am not seeing where this is getting used.

Reply:
the location one isn't yet.